app/assets/stylesheets/inline-embed.scss

```diff
@@ -0,0 +1,15 @@
+.inline-embed {
+  background-color: #dddddd;
```

Comment:
Color literals like `#dddddd` should only be used in variable declarations; they should be referred to via variable everywhere else.